root/static/scripts/relationship-editor/utility/getOpenEditsLink.js

```diff
@@ -8,15 +8,28 @@
  */
 
 import isDatabaseRowId from '../../common/utility/isDatabaseRowId.js';
+import type {
+  LinkRelationshipT,
+} from '../../edit/externalLinks.js';
 import type {
   RelationshipStateT,
 } from '../types.js';
 
+
```

Comment:
is adding two newlines after the all the imports something you want to enforce with eslint? I don't mind either way but noticed you tend to add it

Reply:
Huh. I'm not sure I even do it consciously - I don't know if it's better and I might even rather enforce just one and stop me from doing this :D

Reply:
I removed the extra lines here - seems https://github.com/import-js/eslint-plugin-import/pull/1933 has been there for three years but is unmerged, so there's currently no `exact` option. Not sure if there's something we can do but I commented anyway.